document/03-installing/03-keycloak.tex

```diff
@@ -0,0 +1,45 @@
+\subsection{Installing Keycloak}\label{subsec:installing-keycloak}
+
+\emph{Note: It is preferred to use an existing instance of Keycloak
+if possible.}
+
+Information about Keycloak installation is available at:
+\url{https://www.keycloak.org/documentation.html}.
+
+
+\subsection{Configuring Keycloak}\label{subsec:configuring-keycloak}
+
+Papaya requires:
+\begin{itemize}
+    \item a client with ID \texttt{papaya-web},
+    \item a user with name \texttt{papaya-cli}
+    and role \texttt{realm-management/view-users}.
+\end{itemize}
+
+\subsubsection*{Configure \texttt{papaya-web}}
+
+\begin{enumerate}
+    \setlength\itemsep{0em}
```

Comment:
Takie podpunkciki bym chyba dała wszystkie wielką albo wszystkie małą

Reply:
Jak już literowe podpunkty, to w tym przypadku raczej małymi.